### PR TITLE
build: updates arrow indicator cell adds alt token icons

### PR DESCRIPTION
**Pull request by matthew-garrett**

### CODE REVIEW

File: packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx [hunk 1]
```diff
@@ -33,18 +34,15 @@ export const ArrowIndicatorCell = ({
...
-  let backgroundColor = palette.info.light;
-  // Don't show background highlight on default
-  if (arrowUp === null) backgroundColor = palette.background.default;
-  if (arrowUp === true && greenOnArrowUp === false)
-    backgroundColor = palette.error.light;
-  if (arrowUp === false && greenOnArrowUp === true)
-    backgroundColor = palette.error.light;
-  const valueTextColor = isNegative
+
+  const valueTextColor = tooRisky
```
Comment (jeffywu) [inline packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx:43]: The nested ternery here is hard to read. What about  `tooRisky || isNegative ? palette.error.main : ...`

Comment (matthew-garrett) [reply packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx:43]: Can do

File: packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx [hunk 2]
```diff
@@ -33,18 +34,15 @@ export const ArrowIndicatorCell = ({
     },
   } = cell;
   const { palette, shape, spacing } = useTheme();
-  const colorOnUp = greenOnArrowUp ? palette.primary.light : palette.error.main;
-  const colorOnDown = greenOnArrowUp
+  const arrowColor = tooRisky
     ? palette.error.main
+    : greenOnArrowUp
```
Comment (jeffywu) [inline packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx:39]: You still need to maintain the logic in the if block below. So:

greenOnArrowUp && arrowUp => green
greenOnArrowUp && !arrowUp => orange
!greenOnArrowUp && arrowUp => orange
!greenOnArrowUp && !arrowUp => green

Comment (jeffywu) [reply packages/common/mui/src/lib/data-table-cells/arrow-indicator-cell/arrow-indicator-cell.tsx:39]: Or basically, you can add back the `colorOnUp` and `colorOnDown` variables back in.

File: packages/shared/trade/src/transaction-sidebar/components/liquidation-risk.tsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ export const LiquidationRisk = ({ state }: { state: TradeState }) => {
     columns.push({
       Header: <FormattedMessage defaultMessage={'Updated'} />,
       Cell: ArrowIndicatorCell,
+      tooRisky,
```
Comment (jeffywu) [inline packages/shared/trade/src/transaction-sidebar/components/liquidation-risk.tsx:44]: This also need to be on the PositionDetailsTable

**Pull request merged**